doc/api/util.md

```diff
@@ -71,6 +71,25 @@ callbackFunction((err, ret) => {
 });
 ```
 
+## `util.colorText(format, text)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `format` {string} A color format defined in `util.inspect.colors`.
```

Comment:
Accepting `string | string[]` would be really helpful to e.g. make text both a certain color and underline/italic/bold/... at the same time